core/reporter-api/src/main/java/org/arquillian/reporter/api/model/entry/FileEntry.java

```diff
@@ -1,9 +1,61 @@
 package org.arquillian.reporter.api.model.entry;
 
+import java.io.File;
+import java.nio.file.Path;
+
 /**
  * An {@link Entry} representing a file
  *
  * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
  */
 public class FileEntry implements Entry {
+
+    private String path;
```

Comment:
I would call it `filePath` to add more semantic - when `path` is used then the resulting JSON contains:
`
"entries": [
      {
        "path": "pom.xml"
      }
    ],
`
which is really not self-explaning...